src/jemalloc.c

```diff
@@ -2384,26 +2384,19 @@ imalloc_sample(static_opts_t *sopts, dynamic_opts_t *dopts, tsd_t *tsd,
     size_t usize, szind_t ind) {
 	void *ret;
 
-	/*
-	 * For small allocations, sampling bumps the usize.  If so, we allocate
-	 * from the ind_large bucket.
-	 */
-	szind_t ind_large;
-
 	dopts->alignment = prof_sample_align(dopts->alignment);
```

Comment:
I forgot to mention -- `prof_sample_align` needs to be updated as well, since it assumed using large size classes (which are all page aligned, outside of the `cache_oblivious` / addr randomization case). This is also why the bumped_size was using s2u instead of sa2u (which would make no difference). Now we should be enforcing that explicitly.